configure.ac

```diff
@@ -53,7 +53,7 @@ AC_ARG_WITH(nopylint,
 
 [if test "$with_nopylint" = "no"]
 [then]
-    AC_PATH_PROG([PYLINT], [python3-pylint], [no])
+    AC_PATH_PROG([PYLINT], [pylint-3], [no])
```

Comment:
I would rather create a new f25 branche in github for f25 and older Fedoras and do renaming of the pylint only on master branche.